### PR TITLE
[WIP] Updates for ephemeral sandboxes.

### PR DESCRIPTION
**Pull request by jmcarp**
* Use a single sandbox org called "sandbox"
* Set space names to user email
* Drop cache of updated users

### CODE REVIEW

**Comment by jmcarp** [comment]
@jcscottiii, since you've been reviewing the go patches for this feature, want to take a look at this? It's ready for review but not merge.

**Comment by jcscottiii** [comment]
LGTM. Glad you did `Set space names to user email` to prevent collisions of same username between agencies :)

**Comment by jmcarp** [comment]
@mogul: when should we start putting sandboxes in the shared `sandbox` org with spaces named by username?

**Comment by jmcarp** [comment]
@jcscottiii: based on conversation at / after standup, I think we can merge this once we're happy with the code and once https://github.com/18F/cg-site/pull/762 is ready to go.

cc @mogul

File: monitor_helper.rb [hunk 1]
```diff
@@ -1,42 +1,22 @@
...
-		environment
-
-	end
-
+  def is_valid_email(username)
+    !username.nil? && !username.index("@").nil?
+  end
+
+  def is_whitelisted_email(email)
+    email.end_with?('.gov', '.mil')
```
Comment (LinuxBozo) [inline monitor_helper.rb]: This needs to include `.fed.us`

**Comment by jmcarp** [comment]
Updated to include .fed.us emails.

**Comment by cnelson** [comment]
I noticed the sandbox bot currently requires admin access to CF which makes me nervous.  Now that it doesn't need to provision orgs on demand, WDYT about moving org creation to https://github.com/18F/cg-sandbox so we can drop the sandbox bot's CF permissions to just OrgManager on that org, and scim.read in UAA ?

**Comment by jmcarp** [comment]
Sounds good--how about immediately recreating the sandbox org after deleting it in https://github.com/18F/cg-sandbox/blob/master/cmd/purge/main.go? I'm guessing we'd also need the go script to restore the ruby script's user's org role at that point.

**Comment by cnelson** [comment]
SGTM

**Comment by cnelson** [comment]
I merged in the recent changes from staging to this branch, could probably use some 👀 to make sure I didn't break anything

**Pull request closed without merging**